### ISSUE TITLE
pgBouncer not immediately available when resetting

### ISSUE THREAD

**Issue by mr-bjerre** [supabase/cli#82]
# Bug report

## Describe the bug

When you run `supabase db reset` then pgBouncer is not immediately available. 

This is an issue if you want to run a seed script that uses the pgbouncer.

**Comment by KirioXX** [comment]
I have a similar issue but in my case, it prevents the local server to start.
When I try to connect to the database  I get this error:
```sh
psql: error: connection to server at "localhost" (::1), port 54322 failed: ERROR:  pgbouncer cannot connect to server
```
This occurs to me since I updated the cli to version 0.13.0 and refreshed all images to their latest version.
This is the list of my current images:
```
REPOSITORY                                      TAG                CREATED         
supabase/realtime                              v0.18.0       		5 days ago      
supabase/postgres-meta	                v0.28.0        	6 days ago      
supabase/gotrue								v2.2.6         	6 days ago      
supabase/pgadmin-schema-diff		cli-0.0.4      	2 weeks ago     
edoburu/pgbouncer							1.15.0         		2 months ago    
supabase/storage-api						v0.9.3        		3 months ago    
supabase/postgres							13.3.0         		4 months ago    
postgrest/postgrest							v8.0.0         	4 months ago    
kong													2.1            		7 months ago    
```

**Comment by soedirgo** [comment]
Thanks filing this - we can modify `supabase db reset` to wait for the db to be ready before it returns.

**Comment by benjamin-kirkbride** [comment]
This is really screwing me up as well, any ETA on this one?

**Comment by soedirgo** [comment]
I'd say next week, currently looking at other issues :pray:

**Comment by soedirgo** [comment]
This should now be fixed with the removal of pgbouncer in v0.15.11.
